### PR TITLE
Make further changes to the build a web app to use a Machine Learning model section

### PR DESCRIPTION
**Pull request by LongfeiH**
# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How has this been tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] [local build the book](https://github.com/open-academy/machine-learning/blob/main/CONTRIBUTING.md#build-the-book)
- [ ] review the artifacts from local build
- [ ] [local build the slides](https://github.com/open-academy/machine-learning/blob/main/CONTRIBUTING.md#build-the-slides-optional) if appliable
- [ ] Unit test if appliable

# Checklist:

- [ ] My code follows the [style guidelines](https://github.com/open-academy/machine-learning/blob/main/STYLE_GUIDE.md) of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have applied tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: open-machine-learning-jupyter-book/ml-fundamentals/build-a-web-app-to-use-a-machine-learning-model.md [hunk 1]
```diff
@@ -335,18 +335,16 @@ On the `/predict` route, several things happen when the form is posted:
 
 Using a model this way, with Flask and a pickled model, is relatively straightforward. The hardest thing is to understand what shape the data is that must be sent to the model to get a prediction. That all depends on how the model was trained. This one has three data points to be input in order to get a prediction.
 
-In a professional setting, you can see how good communication is necessary between the folks who train the model and those who consume it in a web or mobile app. In our case, it's only one person, you!
-
 ---
```
Comment (zhangqi444) [inline open-machine-learning-jupyter-book/ml-fundamentals/build-a-web-app-to-use-a-machine-learning-model.md:338]: remove this.

File: open-machine-learning-jupyter-book/ml-fundamentals/build-a-web-app-to-use-a-machine-learning-model.md [hunk 2]
```diff
@@ -150,10 +150,10 @@ Now, it's time to _pickle_ your model! You can do that in a few lines of code. O
 ```{code-cell}
 import pickle
```
Comment (zhangqi444) [inline open-machine-learning-jupyter-book/ml-fundamentals/build-a-web-app-to-use-a-machine-learning-model.md:151]: add an empty line after the import.

File: open-machine-learning-jupyter-book/ml-fundamentals/build-a-web-app-to-use-a-machine-learning-model.md [hunk 3]
```diff
@@ -150,10 +150,10 @@ Now, it's time to _pickle_ your model! You can do that in a few lines of code. O
...
 model_filename = 'ufo-model.pkl'
-pickle.dump(model, open(model_filename,'wb'))
+pickle.dump(model, open(model_filename, 'wb'))
 
-model = pickle.load(open('ufo-model.pkl','rb'))
-print(model.predict(pd.DataFrame([[50,44,-12]])))
+model = pickle.load(open('ufo-model.pkl', 'rb'))
```
Comment (zhangqi444) [inline open-machine-learning-jupyter-book/ml-fundamentals/build-a-web-app-to-use-a-machine-learning-model.md:155]: Is this the right path? Can you make sure your code is successfully built?

**Review by zhangqi444 — APPROVED** [review]

File: open-machine-learning-jupyter-book/ml-fundamentals/build-a-web-app-to-use-a-machine-learning-model.md (hunk 1 above)
Comment (LongfeiH) [reply open-machine-learning-jupyter-book/ml-fundamentals/build-a-web-app-to-use-a-machine-learning-model.md:338]: This has been removed

File: open-machine-learning-jupyter-book/ml-fundamentals/build-a-web-app-to-use-a-machine-learning-model.md (hunk 3 above)
Comment (LongfeiH) [reply open-machine-learning-jupyter-book/ml-fundamentals/build-a-web-app-to-use-a-machine-learning-model.md:155]: This change has been implemented in the latest version of open-academy

Comment (LongfeiH) [reply open-machine-learning-jupyter-book/ml-fundamentals/build-a-web-app-to-use-a-machine-learning-model.md:155]: The path has been modified to ../assets/pickle/ufo-model.pkl，so I didn't make any more changes

File: open-machine-learning-jupyter-book/ml-fundamentals/build-a-web-app-to-use-a-machine-learning-model.md (hunk 1 above)
Comment (zhangqi444) [reply open-machine-learning-jupyter-book/ml-fundamentals/build-a-web-app-to-use-a-machine-learning-model.md:338]: Could you please double check? Let's make sure all the comments are addressed.

**Pull request merged**